public/app/features/profile/UserProfileEditForm.tsx

```diff
@@ -22,37 +23,57 @@ export const UserProfileEditForm: FC<Props> = ({ user, isSavingUser, updateProfi
     <Form onSubmit={onSubmitProfileUpdate} validateOn="onBlur">
       {({ register, errors }) => {
         return (
-          <FieldSet label={<Trans id="edit-user-profile.title">Edit profile</Trans>}>
-            <Field label="Name" invalid={!!errors.name} error="Name is required" disabled={disableLoginForm}>
+          <FieldSet label={<Trans id="user-profile.title">Edit profile</Trans>}>
+            <Field
+              label={t({ id: 'user-profile.fields.name-label', message: 'Name' })}
```

Comment:
I would much prefer to do 

```typescript
            <Field
              label={<Trans id="user-profile.fields.name-label">Name</Trans>}
```

However, the `label` prop here treats it different if you pass in ReactNode vs string.

I would prefer to use `<Trans />` everywhere except when a string is strictly required (like `placeholder` on inputs) where we use `t()` instead. I find `<Trans />` easier to read than all the jumble of brackets from `label={t({})}`.

Reply:
i don't mind either syntax so much tbh. the only thing i'm not super keen on is the function being called `t`. i guess it's for brevity, but the component isn't called `<T />` so idk why they made the function `t()`? i know we can change it on import, but that relies on every developer remembering to do that 😬

Reply:
I see what you mean, and while yes "one letter variables" are not preferred, I personally don't have much of a problem with a very specific known one letter function.

Either way, I think it we could automate and/or lint whatever we do here.